fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -512,6 +515,13 @@ public void write(DataOutput out) throws IOException {
             }
         }
 
+        // write encryptKeys
+        out.writeInt(name2EncryptKey.size());
```

Comment:
create a new class to contains this `name2EncryptKey` for better extension in future.
And using GSON to serde this new class.